src/auto/AutoModule.lua

```diff
@@ -55,11 +55,13 @@ return function(moduleName)
       end
 
       function module:accGradParameters(input, gradOutput, scale)
-         if not self.grads then
-            self.grads = self.b({input=input, weight=self.weight, bias=self.bias}, gradOutput)
+         if self.weight then
```

Comment:
Without this even creating a simple transfer AutoModule fails, so I'd rather merge this and add tests in another PR. Will take a look when I have time, just started using autograd.